examples/credential-registry-storage-contract/src/lib.rs

```diff
@@ -51,16 +49,16 @@ use concordium_std::*;
 
 const SIGNATURE_DOMAIN: &str = "WEB3ID:STORE";
 
-/// Metadata is a string of two bytes that contain information needed for
+/// Version is a string of two bytes that contain information needed for
```

Comment:
The version will be a number, isn't it? The current comment is a leftover from the `mySome.id` repo where the metadata was a string of two bytes of the `platform` e.g. `LI` for `LinkedIn`.

We could update the comment above a bit, e.g. `Version number to decode the credentials`.

Reply:
I updated the comment and also fixed a test, making it more clear that it's a number and not an encoding of two ASCII chars.